Cython/Compiler/ParseTreeTransforms.py

```diff
@@ -2961,7 +2961,9 @@ def _visit_scoped_children(self, node, gil_state):
             self.visitchildren(node, outer_attrs)
 
         self.nogil = gil_state
-        self.visitchildren(node, exclude=outer_attrs)
+        # FIXME: attrs needs to be passed explicitly to avoid "C function call is missing argument 'attrs'".
+        # This looks like a bug
+        self.visitchildren(node, attrs=None, exclude=outer_attrs)
```

Comment:
This is really a C function call, not a Python method call. We map keyword arguments of C function calls left-to-right, and the `.pxd` declarations do not have a default argument for optional argument that we could pass here.

Not sure if this should really be improved (and if "improving" it would really be an improvement.)
```suggestion
        self.visitchildren(node, attrs=None, exclude=outer_attrs)
```